src/main.cpp

```diff
@@ -86,6 +93,37 @@ void onExitSignal(int signum) {
 }
 
 
+bool setLockFile(QString configPath) {
+
+    QDir configDir(configPath);
+
+    if (! configDir.mkpath(".")) {
+        qFatal("Could not create config file.");
+        exit(EXIT_FAILURE);
+    }
+
+    lockFile = new QLockFile(configDir.absoluteFilePath(".lock"));
+    lockFile->tryLock(0);
+
+    switch (lockFile->error()) {
+        case QLockFile::NoError: {
+            return true;
+        }
+        case QLockFile::LockFailedError: {
```

Comment:
No need for the `case` braces, unless you're declaring variables inside a `case`